diff --git a/extractors/xhmm_ismir.py b/extractors/xhmm_ismir.py
index 61287e4..5800426 100644
--- a/extractors/xhmm_ismir.py
+++ b/extractors/xhmm_ismir.py
@@ -57,7 +57,7 @@ class XHMMDecoder():
                 assert(array[0]>=0)
                 result_names.append(name)
                 result_array.append(list(array))
-        result_array=np.array(result_array,dtype=np.int)
+        result_array=np.array(result_array,dtype=int)
         result_array[:,1]+=1 # bass adjust
         result_logprob=np.log(prob_triad[:,result_array[:,0]])
         bass_collect=result_array[:,1]>=0
@@ -95,7 +95,7 @@ class XHMMDecoder():
             if(is_in_range):
                 assert(array[0]>=0)
                 result_array.append(list(array))
-        result_array=np.array(result_array,dtype=np.int)
+        result_array=np.array(result_array,dtype=int)
         result_array[:,1]+=1 # bass adjust
         result_logprob=np.log(prob_triad[:,result_array[:,0]])
         bass_collect=result_array[:,1]>=0
@@ -109,8 +109,8 @@ class XHMMDecoder():
         n_chord=result_logprob.shape[1]
         dp=np.zeros_like(result_logprob)
         dp[0,1:]-=np.inf
-        dp_max_at=np.zeros((n_frame),dtype=np.int)
-        pre=np.zeros_like(result_logprob,dtype=np.int)
+        dp_max_at=np.zeros((n_frame),dtype=int)
+        pre=np.zeros_like(result_logprob,dtype=int)
         dp[0,:]+=result_logprob[0,:]
         dp_max_at[0]=np.argmax(dp[0,:])
         pre[0,:]=-1
@@ -139,8 +139,8 @@ class XHMMDecoder():
         n_chord=triad_logprob.shape[1]
         dp=np.zeros_like(triad_logprob)
         dp[0,1:]-=np.inf
-        dp_max_at=np.zeros((n_frame),dtype=np.int)
-        pre=np.zeros_like(triad_logprob,dtype=np.int)
+        dp_max_at=np.zeros((n_frame),dtype=int)
+        pre=np.zeros_like(triad_logprob,dtype=int)
         dp[0,:]+=triad_logprob[0,:]
         dp_max_at[0]=np.argmax(dp[0,:])
         pre[0,:]=-1
